### PR TITLE
♾ Universal Signin Page

### LINKED ISSUE #274: Sign in for different users
#### Description
Change sign in to the following userflow
image
- This flow should allow someone to use the same email for mentee, mentor, and admin

### PR DESCRIPTION
**Pull request by Leundai**
## Description :star2:

Fixes #274 
- created prompt for type of account login 
- connected to the homepage
- left the older login page for future reference temporarily

## Screenshots
image
 
## TODOs :star:
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: frontend/src/components/css/SelectLogin.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+.select-image {
+  width: 23vw;
+}
+
+.select-text {
+  text-align: center;
+  color: #a58123;
+  font-size: 2em;
+  font-weight: bold;
+  margin-top: -15%;
```
Comment (Leundai) [inline frontend/src/components/css/SelectLogin.scss:48]: lmk if this is something that is okay to do for css!

Comment (Leundai) [reply frontend/src/components/css/SelectLogin.scss:48]: Specifically the margin-top: -15%

**Review by AngelaLuo49021 — APPROVED** [review]
Looks good!

File: frontend/src/components/css/SelectLogin.scss (hunk 1 above)
Comment (kenetec) [reply frontend/src/components/css/SelectLogin.scss:48]: yaa prolly fine,, wouldn't this be equivalent to `margin-bottom: 15%` though?

File: frontend/src/components/pages/Login.js [hunk 2]
```diff
@@ -93,62 +87,46 @@ function Login() {
...
               }}
             />
           </div>
-          <div className="login-register-container">
-            <div>Don&#39;t have an account?</div>
+          <div className="register-link">
+            Don't Have an account?{" "}
             <NavLink to="/register" className="login-register-link">
               Register
             </NavLink>
```
Comment (kenetec) [inline frontend/src/components/pages/Login.js:127]: The `Register` page can take in an `as` query to denote which kind of user to register as (based on the enum `ACCOUNT_TYPE`). I think you should reflect this here

File: frontend/src/components/pages/Login.js [hunk 3]
```diff
@@ -93,62 +87,46 @@ function Login() {
...
-                  setErrorMessage(
-                    LOGIN_ERROR_MSGS.INCORRECT_NAME_PASSWORD_ERROR_MSG
-                  );
-                  setError(true);
-                } else if (res.result.passwordReset) {
-                  setErrorMessage(LOGIN_ERROR_MSGS.RESET_PASSWORD_ERROR_MSG);
-                  setError(true);
-                } else if (res.result.recreateAccount) {
-                  setErrorMessage(LOGIN_ERROR_MSGS.RECREATE_ACCOUNT_ERROR_MSG);
-                  setError(true);
```
Comment (kenetec) [inline frontend/src/components/pages/Login.js:110]: These error messages give a bit more context to the user as sometimes an email might be sent to reset password. It's totally cool to redo they are set but I think they should still appear to the user.

**Review by kenetec — CHANGES_REQUESTED** [review]
Awesome job!! 🥵 There's a few changes that I think should be made, lemme know if you have questions for any of them!

File: frontend/src/components/css/SelectLogin.scss (hunk 1 above)
Comment (Leundai) [reply frontend/src/components/css/SelectLogin.scss:48]: That didn't work :(

**Review by kenetec — APPROVED** [review]
GRavy 👶🏿

**Pull request merged**